correios/utils.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, *ranges):
 
             try:
                 element = element or [range(*r)]
-            except:
+            except Exception:
```

Comment:
nope. What kind of exception this code can cause? TypeError? IndexError? ValueError?

Reply:
Mando em outro PR a melhoria no tratamento dessa Exception.
👍